packages/shared/src/logger/scopes/update/scenes/firmware.ts

```
@@ -0,0 +1,30 @@
+  @LogToLocal()
+  public updateBootloader(params: {
+    connectType: 'usb' | 'ble';
+    deviceType: string;
+    firmwareVersion: string;
+    targetVersion: string;
+    success: boolean;
+    error?: any;
+  }) {
+    return params;
+  }
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)